docs/en/tutorials/how_to.md

```diff
@@ -0,0 +1,60 @@
+# Tutorial 11: Collection of common usage
```

Comment:
Tutorial 11: How to xxx

This tutorial collects answers to any `How to xxx with MMDetection`. Feel free to update this doc if you meet new questions about `How to` and find the answers!